source/blog/2018-07-13-the-ember-times-issue-55.md

```diff
@@ -36,9 +36,22 @@ Check out what's been going in Emberland 🐹 these days:
 
 ---
 
-## [SECTION TITLE](#section-url)
-
-
+## [Find Ember on the Interwebs with the Power of SEO 🔎](#section-url)
+
+These days, weeks and months lots of efforts have been focussing on improving the
+discoverability of official Ember resources including the website, the Guides or the API docs.
+The visibility of these sites to various search engines 🔍 as e.g. DuckDuckGo, Baidu, AOL Search, Dogpile or Google
+plays a major role in how easy it is for developers to find the information they are looking for.
+To improve Ember's visibility on the web, Search-Engine Optimization (SEO) of these pages is one of the most effective measures.
+
+Simple, yet effective improvements for Ember sites included tests of functional redirects ([1](https://github.com/emberjs/website/pull/3394)) and fixes for broken links ([2](https://github.com/emberjs/website/pull/3271), [3](https://github.com/emberjs/website/pull/3183)), [4](https://github.com/emberjs/website/pull/3182)).
+Other high-impact efforts included a [new meta data structure for the new Guides](https://github.com/ember-learn/guides-app/pull/135)
+and the continuous evaluation of web crawler data.
+
+If you have previous experience with SEO and you're interested in helping to cast some keyword magic ✨🎩 for Ember sites,
```

Comment:
For some reason this renders large text and bold for me...

Reply:
Good catch! It seems a line break to the subsequent `---` and next header title `## Contributors' Corner` was missing here